### ISSUE TITLE
Spec description for cc.default_quota_definition wrong

### ISSUE THREAD

**Issue by JesseTAlford** [cloudfoundry/capi-release#20]
The spec files for the various CC jobs all reference `cc.default_quota_definition` and give the description as `"Local to use a local (NFS) file system.  AWS to use AWS."`.

This is obviously not correct, and should be improved to be meaningful.

Some useful context: the spiff-based manifest-generation approach has reduced people's reliance on spec files. I suspect that as we move away from it (and are thus trying to configure releases based on the spec files themselves), more issues with the legibility and accuracy of spec files will tend to arise.

**Comment by adowns01** [comment]
Hi @JesseTAlford, 

We fixed this in commit b32db09. The spec description is now 
```
The name of the quota definition CC will fallback on for org and space limits from the list of quota definitions
```

Thanks, 
@adowns01 && @youreddy

**Comment by aashah** [comment]
Since there hasn't been any activity, I'm going to close this issue. Please feel free to re-open the issue if there's further information, or if we can help with anything else for this!

- @aashah
